stock_pack_wizard/data/product.ul.csv

```diff
@@ -0,0 +1,32 @@
+id,name,high,width,long,type
+pack_1, SPARK-1/43-12 - SPARK新包装中箱, 23.000 , 18.000 , 27.000,pack
```

Comment:
This file needs either to be translated to english or moved to a separate module part of a chinese localization.